### PR TITLE
Rofi sometimes waits for stdin

### PR DESCRIPTION
**Pull request by desdic**
Small patch for rofi sometimes waits for stdin. This small patch is used by go-pass and others who interacts with rofi

### CODE REVIEW

File: bwmenu [hunk 1]
```diff
@@ -49,7 +49,7 @@ DIR="$(dirname "$(readlink -f "$0")")"
 source "$DIR/lib-bwmenu"
 
 ask_password() {
-  mpw=$(rofi -dmenu -p "Master Password" -password -lines 0) || exit $?
+  mpw=$(printf ''|rofi -dmenu -p "Master Password" -password -lines 0) || exit $?
```
Comment (Mange) [inline bwmenu]: I'd place spaces around the pipe character here.

```
mpw=$(printf '' | rofi -dmenu …)
```

**Review by Mange — CHANGES_REQUESTED** [review]

File: bwmenu (hunk 1 above)
Comment (desdic) [reply bwmenu]: Updated :)

**Comment by mattydebie** [comment]
@desdic   If you could squash these 2 commits, I'll merge them!

**Comment by mattydebie** [comment]
And maybe add a space after the pipe as well while you're at it :grin:

**Comment by desdic** [comment]
Ill do a proper pull request :) it seems that I tried to use github.com's inter face it .. hang on :)

**Pull request closed without merging**